### ISSUE TITLE
Alt-tab breaks mouse clicks on Ubuntu

### ISSUE THREAD

**Issue by HDragonHR** [ppy/osu-framework#2776]
**Describe the bug:** After alt-tabbing from osu-lazer, mouse clicks are sent to the desktop instead of the game. Only the top and left area is affected

**Screenshots or videos showing encountered issue:** https://streamable.com/q1ezs

**osu!lazer version:** local debug
**Ubuntu version:** Ubuntu 19.04

**Logs:** 
[database.log](https://github.com/ppy/osu/files/3510315/database.log)
[performance.log](https://github.com/ppy/osu/files/3510316/performance.log)
[runtime.log](https://github.com/ppy/osu/files/3510317/runtime.log)

**Comment by UselessToucan** [comment]
Can you reproduce the problem using osu-framework visual-tests?

**Comment by HDragonHR** [comment]
Yes, I managed to reproduce it in ```osu-framwork visual-tests```. Of course, I had to set it to fullscreen for the bug to occur. Weird thing is that it took 2 or 3 rounds of cycling through Platform -> Fullscreen and Platform -> Borderless tests. 

![Screenshot from 2019-08-20 16-08-51](https://user-images.githubusercontent.com/43511642/63329616-dc333480-c364-11e9-9d1d-48eae45d92bc.png)

**Comment by karpov-stas** [comment]
Can confirm this on Arch.

**Comment by HDragonHR** [comment]
> Can confirm this on Arch.

What desktop environment are you on?

**Comment by HDragonHR** [comment]
I found a temporary solution for osu.Desktop. 
Set ```Confine mouse cursor to window``` to ```Always```
Setting ```Never``` and ```Fullscreen``` results in the same bug. 

Can't confirm on osu-framework as there is no way to set the confine mode and searching tests for confine does not give any results.

**Comment by UselessToucan** [comment]
> Can't confirm on osu-framework as there is no way to set the confine mode

You can create new TestScene and copy-paste [this](https://github.com/ppy/osu/blob/3f9a7b887373c34be99bce687a1cd25469ac30bd/osu.Game/Overlays/Settings/Sections/Input/MouseSettings.cs#L42) combobox there. Note that you will need to use basic implementation instead of osu! styled one.

**Comment by HDragonHR** [comment]
I dont follow/understand your steps @UselessToucan 

Instead I found ```Input -> InputManager -> ToggleConfineMouse``` in osu.Framework.Tests.

Here the bug did not appear.

**Comment by HDragonHR** [comment]
I have switched to Manjaro KDE 18.0.4 , and the bug is gone.
Now it seems to be a GNOME issue?

And I just realised, that the context menu in my screenshot is a weird context menu. The context menu is way shorter than anything I have launched around GNOME.
